route/handler.go

```diff
@@ -1,19 +1,21 @@
 package route
 
 import (
-	"github.com/gorilla/mux"
-
 	"github.com/rodrigomkd/go-rest-api/controller"
+
+	"github.com/gorilla/mux"
 )
 
 // HandlerRequests - Returns router that handles API paths
-func GetRouter() *mux.Router {
+func GetRouter(c controller.Controller) *mux.Router {
 	// creates a new instance of a mux router
 	router := mux.NewRouter().StrictSlash(true)
 
 	api := router.PathPrefix("/api/v1").Subrouter()
-	api.HandleFunc("/items", controller.GetItems).Methods("GET")
-	api.HandleFunc("/items/{id}", controller.GetItem).Methods("GET")
+
+	api.HandleFunc("/items", c.GetItems).Methods("GET")
```

Comment:
You could replace the `GET` string by the http constant `http.MethodGet` I think it was